backend/Application/Api/Rest/ExportController.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.Threading.Tasks;
+using System.Text;
+using Application.Api.GraphQL.Accounts;
+using Application.Api.GraphQL.EfCore;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace Application.Api.Rest;
+
+[ApiController]
+public class ExportController : ControllerBase
+{
+    private readonly IDbContextFactory<GraphQlDbContext> _dbContextFactory;
+
+    public ExportController(IDbContextFactory<GraphQlDbContext> dbContextFactory)
+    {
+        _dbContextFactory = dbContextFactory;
+    }
+
+    [HttpGet]
+    [Route("rest/export/statement")]
+    public async Task<ActionResult> GetStatementExport(string accountAddress)
+    {
+        await using var dbContext = await _dbContextFactory.CreateDbContextAsync();
+
+        if (!ConcordiumSdk.Types.AccountAddress.TryParse(accountAddress, out var parsed))
+        {
+            return BadRequest("invalid account format");
+        }
+
+        var baseAddress = new AccountAddress(parsed!.GetBaseAddress().AsString);
+        var account = dbContext.Accounts
+            .AsNoTracking()
+            .SingleOrDefault(account => account.BaseAddress == baseAddress);
+        if (account == null)
+        {
+            return NotFound("account does not exist");
+        }
+
+        var query = dbContext.AccountStatementEntries
+            .AsNoTracking()
+            .Where(x => x.AccountId == account.Id);
+
+        var scalarQuery = query.Select(x => new
+        {
+            x.Timestamp,
+            x.EntryType,
+            x.Amount,
```

Comment:
Is this amount the "balance change" for the given account?

Reply:
yes